### PR TITLE
migrate test_event to pytest

### PR DESCRIPTION
**Pull request by frogunder**
### What does this PR do?

migrate test_event to pytest

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [ ] Docs
- [ ] Changelog - https://docs.saltproject.io/en/master/topics/development/changelog.html
- [x] Tests written/updated

### Commits signed with GPG?
Yes

Please review [Salt's Contributing Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

File: tests/pytests/unit/modules/test_event.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import pytest
+
+import salt.modules.event as event
+import salt.utils.event
+from tests.support.mock import MagicMock, patch
+from tests.support.paths import SALT_CODE_DIR
+
+
+@pytest.fixture
+def configure_loader_modules():
```
Comment (s0undt3ch) [inline tests/pytests/unit/modules/test_event.py:16]: I believe we now have a `minion_opts` fixture available. Let's use that instead.
Ping me for additional clarifications if need be.

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: tests/pytests/unit/modules/test_event.py (hunk 1 above)
Comment (frogunder) [reply tests/pytests/unit/modules/test_event.py]: Hopefully did this right, thanks.

File: tests/pytests/unit/modules/test_event.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+from tests.support.mock import MagicMock, patch
+from tests.support.paths import SALT_CODE_DIR
+
+
+@pytest.fixture
+def minion_opts(minion_opts):
+    minion_opts.update(
+        {
+            "id": "id",
+            "sock_dir": SALT_CODE_DIR,
```
Comment (s0undt3ch) [inline tests/pytests/unit/modules/test_event.py:20]:
```suggestion
```
This is not necessary, and probably transport too.

Comment (s0undt3ch) [reply tests/pytests/unit/modules/test_event.py:20]: Actually, looking at the tests, this whole fixture shouldn't be necessary.

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: tests/pytests/unit/modules/test_event.py (hunk 2 above)
Comment (frogunder) [reply tests/pytests/unit/modules/test_event.py]: Removed `sock_dir` and `transport`. 

`id` seems to be needed for me to get the tests to pass.

Unless I am doing something wrong, which is highly possible. 

Thanks for input.

File: tests/pytests/unit/modules/test_event.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+@pytest.fixture
+def minion_opts(minion_opts):
+    minion_opts.update(
+        {
+            "id": "id",
+        }
+    )
+    return minion_opts
+
+
```
Comment (s0undt3ch) [inline tests/pytests/unit/modules/test_event.py:23]:
```suggestion
```

File: tests/pytests/unit/modules/test_event.py [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+def test_fire_master():
+    """
+    Test for Fire an event off up to the master server
+    """
+    with patch("salt.crypt.SAuth") as salt_crypt_sauth, patch(
+        "salt.channel.client.ReqChannel.factory"
+    ) as salt_transport_channel_factory:
+
+        preload = {
+            "id": "id",
```
Comment (s0undt3ch) [inline tests/pytests/unit/modules/test_event.py:38]:
```suggestion
def test_fire_master(minion_opts):
    """
    Test for Fire an event off up to the master server
    """
    with patch("salt.crypt.SAuth") as salt_crypt_sauth, patch(
        "salt.channel.client.ReqChannel.factory"
    ) as salt_transport_channel_factory:

        preload = {
            "id": minion_opts["id"],
```

**Review by s0undt3ch — CHANGES_REQUESTED** [review]
This should do it.

File: tests/pytests/unit/modules/test_event.py (hunk 3 above)
Comment (frogunder) [reply tests/pytests/unit/modules/test_event.py]: Changed, thanks

File: tests/pytests/unit/modules/test_event.py (hunk 4 above)
Comment (frogunder) [reply tests/pytests/unit/modules/test_event.py:28]: Thanks, I think it should be good now.

**Review by s0undt3ch — APPROVED** [review]

**Review by garethgreenaway — APPROVED** [review]

**Pull request merged**